ipfix/interpret.go

```diff
@@ -77,6 +77,15 @@ func Interpret(b *[]byte, t FieldType, eKey *ElementKey) interface{} {
 		} else {
 			return string(*b)
 		}
+	case Ipv4StringOrString:
+		str := string(*b)
+		// if string parses to an IPV4, return it, else return the string
+		ip := net.ParseIP(str)
+		if ip != nil && ip.To4() != nil {
+			return ip
```

Comment:
I think you need to set `eKey.MultiTypeID = 1` in the case where it's an IP if you want it to end up in a separate key in the output json (so you can have 2 single typed arrays).

Reply:
Good catch, fixed.